src/commands/task/delete.ts

```diff
@@ -41,6 +41,10 @@ export default class TaskDeleteCommand extends Command {
     const taskId = flags[`task-id`]
     const subscriberId = flags[`subscriber-id`]
 
+    if (!taskId && !flags.tag) {
+      this.error(`Must specify either a task id or tag to delete`)
```

Comment:
Should the command exit due to this error?

Reply:
I figured yes since if they didn't specify which task/tasks they wanted to delete it shouldn't/wouldn't be able to delete anything.